internal/statsig/statsig_client.go

```diff
@@ -18,9 +18,18 @@ type Client struct {
 	Client   *http.Client
 }
 
+// ErrorResponse is the representation of the response body when an error occurs. This is different from
+// the APIResponse struct as it only contains the message and status code of the error, rather than the data.
+type ErrorResponse struct {
+	Message    string `json:"message"`
+	StatusCode int    `json:"status"`
+}
+
+// NewClient creates a new Statsig client with the provided API key.
+// The Client instance includes an HTTP client with a 10-second timeout to be used for API requests.
 func NewClient(_ context.Context, apiKey string) (*Client, error) {
 	return &Client{
-		HostURL:  "https://api.statsig.com/console/v1",
+		HostURL:  "https://statsigapi.net/console/v1",
```

Comment:
did their api url change?

Reply:
In their latest official docs, yes. However, they're all compatible and yield the same results